Tests/SwiftParserTest/Expressions.swift

```diff
@@ -467,7 +467,7 @@ final class ExpressionTests: XCTestCase {
                 diagnostics: [
                   DiagnosticSpec(message: "Expected identifier in '#keyPath' expression"),
                   DiagnosticSpec(message: "Expected ')' to end '#keyPath' expression"),
-                  DiagnosticSpec(locationMarker: "MISSING_VALUE", message: "Expected value after ':' in function call"),
+                  DiagnosticSpec(locationMarker: "MISSING_VALUE", message: "Expected value in function call"),
```

Comment:
Ideally this would be "argument" but that's not particularly easy with the current scheme 🤔.